TAScheduler/acceptance_tests/sections/test_create.py

```diff
@@ -37,7 +37,7 @@ def setUp(self):
     def test_adds_to_database(self):
         resp = self.client.post(self.url, {
             'section_code': self.good_code,
-            'course_id': self.course.id,
+            'course_name': self.good_name,
```

Comment:
I think I have misled you on accident, when creating a section the form posts a value called `course_id` which is itself a primary key from the table `Course`.

Reply:
So this would have to return to 

```suggestion
            'course_id': self.course.id,
```

as would the instances below, as well as in `test_edit.py`